src/components/Field/Field.sass

```diff
@@ -1,17 +1,47 @@
 @import ../../styles/variables.sass
 
 $appBarOffset: 135px
+$bottomControlsOffset: 260px
+$horizontalQuickSelectOffest: 80px;
```

Comment:
looks like you don't need this semi-colon here

Reply:
Thanks for spotting this! Fixed in https://github.com/jeremyckahn/farmhand/commit/e3f32839fec032811d0e808dfc43ab30da4fbb14.